src/aks-preview/azext_aks_preview/_validators.py

```diff
@@ -803,3 +803,33 @@ def validate_application_security_groups(namespace):
     for asg in asg_ids.split(","):
         if not is_valid_resource_id(asg):
             raise InvalidArgumentValueError(asg + " is not a valid Azure resource ID.")
+
+
+def validate_utc_offset(namespace):
+    """Validates --utc-offset for aks maintenanceconfiguration add/update commands."""
+    if namespace.utc_offset is None:
+        return
+    utc_offset_regex = re.compile(r'^[+-]\d{2}:\d{2}$')
```

Comment:
Please add specific examples through comments to make the regular expressions easier to understand